js/main.js

```diff
@@ -1,13 +1,25 @@
-import * as draw from './draw.js';
```

Comment:
Interesting - I'm assuming this was the linter? 
since the functions are no longer referred to from their module's namespace, can we rename frame to drawFrame?

Reply:
yep makes sense, will do!